transformers/component_tests/queries/get_block_heights_for_transformed_diffs_test.go

```diff
@@ -0,0 +1,56 @@
+// VulcanizeDB
+// Copyright © 2020 Vulcanize
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package queries
+
+import (
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/test_helpers"
+	"github.com/makerdao/vulcanizedb/libraries/shared/storage"
+	"github.com/makerdao/vulcanizedb/pkg/fakes"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Get Block Height for Transformed Diffs Query", func() {
+	const blockHeightsForTransformedStorageDiffs = `SELECT * FROM api.get_block_heights_for_transformed_diffs()`
+
+	BeforeEach(func() {
+		test_config.CleanTestDB(db)
+	})
+
+	It("ignores untransformed diffs", func() {
+		test_helpers.CreateFakeDiffRecord(db)
+
+		var diff int
```

Comment:
Maybe this should be a `uint64`?

Reply:
Could you say more here? I'd be fine either way but am unsure of the costs/benefits

Reply:
`int` seems to be 32 or 64 bits depending on platform. Since blocks will never be negative and potentially large numbers does it make sense to explicitly have them be uint and 64 bits in size?

Reply:
My brain is mush. This is the test. I was thinking about real world block heights that could need larger size.